components/script/dom/htmlanchorelement.rs

```diff
@@ -667,7 +682,24 @@ pub fn follow_hyperlink(subject: &Element, hyperlink_suffix: Option<String>) {
         };
 
         // Step 14
-        debug!("following hyperlink to {}", url);
-        target_window.load_url(url, replace, false, referrer, referrer_policy);
+        let pipeline_id = target_window.upcast::<GlobalScope>().pipeline_id();
+        println!("Url: {:?}", url);
```

Comment:
`println!` should be `debug!`

Reply:
Yes I'm going to remove those print and console.logs...